docs/api.json

```diff
@@ -0,0 +1,1070 @@
+            "fields": ["String"]
+          },
+          "response": "types.visitorUser || types.volunteerUser || types.adminUser"
+        },
+        "PUT": {
+        "description": "Update details of a specific user",
+        "intendedFor": ["twineAdmin", "fundingBody", "cbAdmin"],
+        "scope": ["users_details-children:write"],
+        "body": {
+          "name": "String",
+          "regionId": "Number",
+          "sectorId": "Number",
+          "organisationId": "Number",
+          "gender": "String",
+          "birthYear": "String",
+          "email": "String",
+          "phoneNumber": "String",
+          "emailConsent": "Boolean",
+          "smsConsent": "Boolean",
+          "qrCode": "String"
+        },
+        "response": "types.visitorUser || types.volunteerUser || types.adminUser"
+        },
+        "DELETE": {
+        "description": "Delete a specific user",
+        "intendedFor": ["twineAdmin", "fundingBody", "cbAdmin"],
+        "scope": ["users_details-children:write"],
+        "body": {
+          "userId": "Number"
+        },
+        "response": null
+        }
+      },
+      "/password/reset": {
+        "POST": {
+          "description": "Reset password for a single user",
+          "auth": true,
+          "intendedFor": ["twineAdmin", "fundingBody", "cbAdmin", "visitor", "volunteer", "volunteerAdmin"],
+          "scope": ["user_details-own:write"],
+          "body": {
+            "userId": "Number",
+            "email": "String",
+            "password": "String",
+            "confirmPassword": "String"
+          },
+          "response": null
+        }
+      },
+      "/admin/me": {
+        "GET": {
+          "description": "Retrieve details of your own admin user account",
+          "auth": true,
+          "intendedFor": ["twineAdmin", "fundingBody", "cbAdmin"],
+          "scope": ["user_details-own:read"],
+          "query": {
+            "fields": ["String"]
+          },
+          "response": "types.adminUser"
+        },
+        "PUT": {
+        "description": "Update details of a your own admin user account",
+        "auth": true,
+        "intendedFor": ["twineAdmin", "fundingBody", "cbAdmin"],
+        "scope": ["user_details-own:write"],
+        "query": {
+          "name": "String",
+          "regionId": "Number",
+          "sectorId": "Number",
+          "organisationId": "Number",
+          "gender": "String",
+          "birthYear": "String",
+          "email": "String",
+          "phoneNumber": "String",
+          "emailConsent": "Boolean",
+          "smsConsent": "Boolean",
+          "delete": "Boolean"
+        },
+        "response": "types.adminUser"
+        },
+        "DELETE": {
+          "description": "Delete your own admin user account",
+          "auth": true,
+          "intendedFor": ["twineAdmin", "fundingBody", "cbAdmin"],
+          "scope": ["user_details-own:delete"],
+          "body": {
+            "userName": "String"
```

Comment:
Why require the user name? As a confirmation step? Maybe the password would make more sense? Or both.